catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/intents/looks/paintroid/PocketPaintNewSpriteDiscardIntentTest.java

```diff
@@ -107,9 +114,48 @@ public void testCancelImport() {
 
 		intended(expectedIntent);
 
+		closeSoftKeyboard();
+
+		onView(withText(newSpriteName))
+				.check(matches(isDisplayed()));
+
+		onView(withText(R.string.new_empty_sprite_note))
+				.check(matches(isDisplayed()));
+
+		onView(withText(R.string.cancel))
+				.perform(click());
+
 		onRecyclerView().checkHasNumberOfItems(1);
 	}
 
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void testAddEmptySprite() {
+		onView(withId(R.id.button_add))
```

Comment:
This test fails locally on my emulator because of a `NullPointerException` (uri is null at `org.catrobat.catroid.io.StorageOperations.copyUriToDir(StorageOperations.java:209` )
However, it seems that the test works on Jenkins, so maybe its just a problem with my emulator.